hubconf.py

```diff
@@ -4,6 +4,4 @@
 # This source code is licensed under the CC-by-NC license found in the
 # LICENSE file in the root directory of this source tree.
 #
-from models import deit_tiny_patch16_224, deit_small_patch16_224, deit_base_patch16_224
```

Comment:
Hi, I think that these line cannot be removed, it's essential for `torch.hub` to be working.

Reply:
As @zhiqwang mentioned, torchhub requires this import to work, so can you keep them?